### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by ulic75** [ulic75/power-flow-card#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to, well, discuss things that maybe shouldn't be "issues" yet. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with others.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by flixlix** [comment]
Hi, I just wanted to thank you for making this card!
Since the release of the HA energy management feature I have been wanting this to be a reality!
Thank you!!!
